package.json

```diff
@@ -33,5 +33,8 @@
     "prettier": "2.5.1",
     "ts-node": "^10.4.0",
     "typescript": "^4.5.4"
+  },
+  "dependencies": {
+    "@api3dao/operations": "api3dao/operations#384840c4c0fa49698a06c15c3225f30993e9e2ff"
```

Comment:
For now, using the @aquarat branch since that contains the info I need https://github.com/api3dao/operations/pull/3/commits/384840c4c0fa49698a06c15c3225f30993e9e2ff but it's not yet ready to be merged.